docs/modules/ROOT/pages/limitations.adoc

```diff
@@ -5,3 +5,42 @@ The Stackable Agent has limitations, some of which are documented here.
 * Kubernetes v1.19+ is required.
 * The maximum number of pods supported by the agent is currently hardcoded to 110.
   This restriction can be lifted if the need arises.
+* SSL certificate is not automatically renewed due to upstream bug https://github.com/krustlet/krustlet/issues/553 (manual activity required described below)
+
+== SSL certificate renewal ==
+
+The agent SSL certificate expiry date can be observed via `openssl x509 -enddate -noout -in /etc/stackable/stackable-agent/secret/agent.crt`.
+
+See upstream documentation about the certificate system at https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/, and the following steps:
+
+To generate a new certificate, first delete any current CSR:
+
+```
+kubectl delete csr/${HOSTNAME}-tls
+```
+
+If you have a CSR already, the agent will fail and log in to the journal (`journalctl -u stackable-agent`):
+
+```
+Sep 09 13:51:59 server1 stackable-agent[15457]: Caused by:
+Sep 09 13:51:59 server1 stackable-agent[15457]:     certificatesigningrequests.certificates.k8s.io "server1-tls" already exists: AlreadyExists
+```
+
+Remove the current certificate file and restart the agent:
+
+```
+sudo rm /etc/stackable/stackable-agent/secret/agent.crt && sudo systemctl restart stackable-agent
+```
+
+And then observe the CSR is pending, with `kubectl get csr`
+
+```
+NAME          AGE   SIGNERNAME                      REQUESTOR    CONDITION
+server1-tls   4s    kubernetes.io/kubelet-serving   kube:admin   Pending
+```
+
+This can be approved with `kubectl certificate approve server1-tls` or you can approve all Pending with:
+
+```
+kubectl get csr -o go-template='{{range .items}}{{if not .status}}{{.metadata.name}}{{"\n"}}{{end}}{{end}}' | xargs --no-run-if-empty kubectl certificate approve
+```
```

Comment:
Approving everything can be a valid approach on test environments but not on production. This documentation is also meant for production usage so we should either add a warning to this paragraph or remove it.

Reply:
Thanks, I'll tweak.